lib-injection/sources/sitecustomize.py

```diff
@@ -203,9 +203,11 @@ def runtime_version_is_supported(python_runtime, python_version):
     supported_versions = RUNTIMES_ALLOW_LIST.get(python_runtime, {})
     if not supported_versions:
         return False
-    return (
-        supported_versions["min"].version <= parse_version(python_version).version < supported_versions["max"].version
-    )
+
+    # Use the next max version as the upper bound
+    max_version = list(supported_versions["max"].version)
+    max_version[-1] += 1
+    return supported_versions["min"].version <= parse_version(python_version).version < tuple(max_version)
```

Comment:
```suggestion
    return supported_versions["min"].version <= parse_version(python_version).version <= tuple(max_version)
```
Would doing this and not modifying the max_version work instead?

Reply:
I don't think it would 🤔  (for example 3.14.1 > 3.14). So I think we would be safer this way.

Reply:
alternative could also be to drop the patch and only check major/minor.

```python
python_version = 
return supported_versions["min"].version[:2] <= parse_version(python_version).version[:2] <= supported_versions["max"].version[:2]
```